shell/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rancher/shell",
-  "version": "3.0.2-rc.6",
+  "version": "3.0.2",
```

Comment:
I am not so sure about bumping a version with a fix, but I do not know what's the normal process for this either.

Reply:
It's needed for a partner extension, as is described on the PR